styles/layout.css

```diff
@@ -0,0 +1,23 @@
+.filter {
+    display: inline-block;
+    padding: 10px;
+    width: auto;
+    font-size: 24px;
+}
+
+.content {
+    float: left;
+    min-width: 90%;
+}
+
+@media screen and (min-width: 400px) {
+    .filter {
+        display: block;
+        float: left;
+    }
+
+    article {
+        min-width: 90%;
+        float:left;
+    }
+}
```

Comment:
Any reason this is being [duplicated](https://github.com/suhk/code-blog/pull/6/files#diff-4d20359eb6c33831836e1dfd6145e9cdR76) in index.css? I don't have much experience with CSS, so I'm curious about this. My gut says there's no reason to duplicate it, but I could be wrong.